extensions/markdown-language-features/src/languageFeatures/references.ts

```diff
@@ -312,30 +312,6 @@ export class MdReferencesProvider extends Disposable {
 	}
 }
 
```

Comment:
This entire file will be deleted once diagnostics are also moved to the language server